### PR TITLE
Add logging to grain in HelloWorld.NetCore

### PR DESCRIPTION
**Pull request by jokin**
I would have find helpful if the new way of logging in the 2.0 version was included in some of the samples.

### CODE REVIEW

**Comment by jokin** [comment]
The Ci build are timeout erros, is there a way to run again without pushing another change?

**Comment by jdom** [comment]
@dotnet-bot test this please
Reason: unrelated test failure

File: Samples/HelloWorld.NetCore/src/HelloWorld.Grains/HelloGrain.cs [hunk 1]
```diff
@@ -8,8 +9,16 @@ namespace HelloWorld.Grains
...
+        private readonly ILogger logger;
+
+        public HelloGrain(ILogger<HelloGrain> logger)
+        {
+            this.logger = logger;
+        }  
+
         Task<string> IHello.SayHello(string greeting)
         {
+            logger.LogWarning($"SayHello message received: greeting = '{greeting}'");
```
Comment (jdom) [inline Samples/HelloWorld.NetCore/src/HelloWorld.Grains/HelloGrain.cs]: I know it's nitpicking, but for a new-comer, receiving a warning might be surprising, they might think something went wrong. Could you change to informational?

Comment (jokin) [reply Samples/HelloWorld.NetCore/src/HelloWorld.Grains/HelloGrain.cs]: Done, I started with Info level but changed to warning to standout between all the messages, but it definitely should be and information level message. Thanks.

**Comment by jdom** [comment]
Thank you @jokin !

**Pull request merged**